core/deployment/src/main/java/io/quarkus/deployment/steps/NativeImageFeatureStep.java

```diff
@@ -284,7 +284,7 @@ public void write(String s, byte[] bytes) {
         if (!resourcePatterns.isEmpty()) {
             // Needed to access LOOKUP_METHOD
             exports.produce(new JPMSExportBuildItem("org.graalvm.nativeimage.base", "com.oracle.svm.util",
-                    GraalVM.Version.VERSION_22_1_0));
+                    GraalVM.Version.CURRENT));
```

Comment:
This should not change. It says that the export should be done for GraalVM >= 22.1